frontend/simple/views/NavBar.vue

```diff
@@ -25,21 +25,22 @@
         </div>
         <div class="nav-center">
           <!-- TODO: use v-for to dynamically generate these? -->
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/new-group'}">Start a group</a>
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/signup'}">New User</a>
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/user'}">Profile</a>
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/user-group'}">Group</a>
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/pay-group'}">Pay Group</a>
-          <a class="nav-item" v-link="{activeClass: 'is-active', path: '/ejs-page'}" id="testEJS">EJS test</a>
+          <!--TODO figure out what needs to be done with active classe after upgrade  "{activeClass: 'is-active', path: '/new-group'}"-->
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="new-group">Start a group</router-link>
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="signup">New User</router-link>
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="user">Profile</router-link>
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="user-group">Group</router-link>
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="pay-group">Pay Group</router-link>
+          <router-link class="nav-item" v-bind:class="{activeClass: 'is-active'}" to="ejs-page" id="testEJS">EJS test</router-link>
```

Comment:
You can get rid of all the `v-bind:class="{activeClass: 'is-active'}"` stuff.

See: `active-class` in: https://router.vuejs.org/en/api/router-link.html